### PR TITLE
Navbar Improvements

### PR DESCRIPTION
**Pull request by ccweaver1**
- Improved styling
- Collapse to topbar on sm and xs screens
- hamburger menu

Note: Not in love with this code... tried to use tailwind's screen sized based class tags, but this still means each nav component is rendered by React even if it's not being displayed... but layout components that export metadata can't be client based components.  Whatever, it works

### CODE REVIEW

File: apps/web/src/app/layout.tsx [hunk 1]
```diff
@@ -24,14 +25,16 @@ export default function BTSHLayout({
...
     <html lang="en">
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
         <div>
-          <BTSHSidebar/>
-          <div className="ml-32 min-h-screen">
+          <BTSHNavbar />
+          {/* tailwind class to push content to the right on lg screens and push down on small screens and xs screens */}
+          <div className="min-h-screen mt-16 md:mt-0 ml-0 md:ml-32">
```
Comment (jmichaliga) [inline apps/web/src/app/layout.tsx:37]: This is typically a tailwind best practice even though you may not love it. :)

File: apps/web/src/components/navigation/useNavigationLinks.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (jmichaliga) [inline apps/web/src/components/navigation/useNavigationLinks.ts:1]: typically the 'use' prefix is reserved for react hook components.

**Review by jmichaliga — APPROVED** [review]
Maybe can refactor to better conventions later, but it looks and works well.

File: apps/web/src/components/navigation/useNavigationLinks.ts (hunk 2 above)
Comment (ccweaver1) [reply apps/web/src/components/navigation/useNavigationLinks.ts:1]: I wanted to leave room for this component to grow into a stateful component that might require a hook (e.g. when the info in the navigation depend on the user profile loaded).

File: apps/web/src/app/layout.tsx (hunk 1 above)
Comment (ccweaver1) [reply apps/web/src/app/layout.tsx:37]: Yeah, the thing that really upsets me here is in the Navbar component both the small and large versions are rendered regardless of screen width.... seems very inefficient to me, but in this case not a big deal

**Comment by ccweaver1** [comment]
Merging because its an improvement, but would rather use some external navbar components in the future.

**Pull request merged**